sdk/textanalytics/ai-text-analytics/samples/v6-beta/typescript/README.md

```diff
@@ -5,7 +5,7 @@ languages:
 products:
   - azure
   - azure-cognitive-services
-  - azure-language-service
+  - language-service
```

Comment:
Please refer to https://github.com/Azure/azure-sdk-for-js/pull/22663#pullrequestreview-1052759310 for my review of the same issue.